lib/hood/kiln.hoon

```diff
@@ -388,33 +381,39 @@
     ==  ==
   ::
   ++  start-sync
-    |=  reset/?
```

Comment:
Why do we no longer need to know whether we're resetting?

Reply:
The only time we were resetting was on the first sync of a star, planet, or moon. I believe we were unconditionally resetting because those ships were booting into kernel without a filesystem and then syncing from their parent. Ie, as soon as they had a full filesystem they did a reset. That's no longer necessary since we're booting into a coherent filesystem that matches the kernel (unless you're freelancing with `-A`).

Reply:
Sounds good.